lib/Screens/account_dashboard.dart

```diff
@@ -20,97 +25,453 @@ class _AccountDashboardState extends State<AccountDashboard> {
+        },
+        leading: const CircleAvatar(),
+        contentPadding: const EdgeInsets.symmetric(),
+        title: TitleText(
+          text: acc.alias,
+          fontSize: 14,
+        ),
+        subtitle: Text(account.accountNumber),
+        trailing: Container(
+          height: 30,
+          width: 60,
+          alignment: Alignment.center,
+          decoration: const BoxDecoration(
+            color: LightColor.lightGrey,
+            borderRadius: BorderRadius.all(
+              Radius.circular(
+                10,
+              ),
+            ),
+          ),
+          child: Text(
+            acc.balance,
+            style: GoogleFonts.muli(
+              fontSize: 12,
+              fontWeight: FontWeight.bold,
+              color: LightColor.navyBlue2,
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  void _showAccountChoosingBottomSheet() {
+    final List<Widget> columnWidgets = [
+      const Padding(
+        padding: EdgeInsets.all(
+          20.0,
+        ),
+        child: TitleText(text: 'Accounts'),
+      ),
+      const Divider(
+        height: 20,
+      ),
+    ];
+    final List<Widget> accountButtons =
+        accounts.map((Account acc) => _getAccountTile(context, acc)).toList();
+
+    columnWidgets.addAll(accountButtons);
+
+    showModalBottomSheet<void>(
+      context: context,
+      builder: (context) {
+        return Container(
+          height: 350.0,
+          color: const Color(0xFF737373),
+          child: Container(
+            decoration: const BoxDecoration(
+              color: Colors.white,
+              borderRadius: BorderRadius.only(
+                topLeft: Radius.circular(
+                  20.0,
+                ),
+                topRight: Radius.circular(
+                  20.0,
+                ),
+              ),
+            ),
+            child: ListView(
+              children: columnWidgets,
+            ),
+          ),
+        );
+      },
+    );
+  }
+
+  Widget _appBar() {
+    return Row(
+      children: <Widget>[
+        const CircleAvatar(),
+        const SizedBox(width: 15),
+        TitleText(
+          text: account.alias,
+        ),
+        const Expanded(
+          child: SizedBox(),
+        ),
+        GestureDetector(
+          onTap: () {
+            _showAccountChoosingBottomSheet();
+          },
+          child: Icon(
+            Icons.short_text,
+            color: Theme.of(context).iconTheme.color,
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _operationsWidget() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceAround,
+      children: <Widget>[
+        _icon(
+          Icons.transfer_within_a_station,
+          'Transfer',
+        ),
+        _icon(
+          Icons.delete_sweep,
+          'Unlink',
+        ),
+        _icon(
+          Icons.code,
+          'Qr Pay',
+        ),
+      ],
+    );
+  }
+
+  Widget _icon(IconData icon, String text) {
```

Comment:
Can you please move this under `lib/widgets` as well, just to make `account_dashboard.dart` cleaner. You can also group the widgets under different folders inside `lib/widgets` if you think this is not really useful in general places